### PR TITLE
My Blog App : Controller specs

### PR DESCRIPTION
**Pull request by ohyajk**
- [x] In this project task, I created the Request spec test files for all Users and Posts controllers.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    end
+
+    it 'should include "Here is a list of users" on the screen' do
+      expect(response.body).to include('Here is a list of users')
+    end
+  end
+
+  describe 'GET #show' do
+    # user = User.create(name: 'Grabrielle', photo: 'mybaby.png', bio: 'A beautiful baby', post_counter: 0)
+    before(:example) { get user_path(2) } # get(:show, params: { id: 2 })
```
Comment (TedLivist) [inline spec/requests/users_spec.rb]: - [x] Kindly remove these commented code lines.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ohyajk,

Good job so far!
✔️ Well written and passing tests
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] **[OPTIONAL]** I would recommend you organize your test files better. You can move these model files to a `spec/models` directory just as you done with the request tests

  image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (ohyajk) [reply spec/requests/users_spec.rb]: Removed from both files. Thanks for letting me know.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ohyajk,

✔️ Clean tests and documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**